src/config.ts

```diff
@@ -101,11 +104,12 @@ export const CORS_CONFIG = {
 export const API_ROUTE = isProduction ? '' : `http://localhost:${PORT}`;
 export const API_DOCS = 'https://docs.igboapi.com';
 
-// IgboSpeech
+// Nkọwa okwu AI Models
 export const SPEECH_TO_TEXT_API = isProduction
   ? 'https://speech.igboapi.com'
   : 'http://localhost:3333';
 
+export const IGBO_TO_ENGLISH_API = ENV_IGBO_TO_ENGLISH_URL;
```

Comment:
nit: can you move this line on line up to line 111? That way there's an empty new line before line 113